version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '6.2.216'
+__version__ = '6.2.255'
```

Comment:
Is there a reason this was bumped to 255? Caught me off guard merging this into my branch.

Reply:
I had to update the version to something unique to appropriately work with the pipeline for the website test integration from TreZ. Not sure what the best practice is here in general, but for future PRs I think I will change this on identical duplicate branches for website testing.

So please feel free to change this to something appropriate when merging.

Reply:
With the latest commit I have reset the version to 6.2.216